kernel/src/main/scala/cats/kernel/CommutativeGroup.scala

```diff
@@ -14,4 +14,14 @@ object CommutativeGroup extends GroupFunctions[CommutativeGroup] {
    * Access an implicit `CommutativeGroup[A]`.
    */
   @inline final def apply[A](implicit ev: CommutativeGroup[A]): CommutativeGroup[A] = ev
+
+  /**
+   * Create a `CommutativeGroup` instance from the given inverse and combine functions and empty value.
+   */
+  @inline def instance[A](emp: A, inv: A => A, cmb: (A, A) => A): CommutativeGroup[A] =
```

Comment:
You probably mean `InvariantSemigroupal` so that we can use `tupledN` right? That's doable but it would more allocations and reduced performance at _runtime_ (going from `(a, b, c, d)` to `(a, (b, (c, d)))` and back) so I'm not sure that would be acceptable for `cats-kernel` which is also used by `albra`.